.travis.yml

```diff
@@ -0,0 +1,57 @@
+sudo: required
+dist: trusty
+notifications:
+  email: false
+language: ruby
+rvm:
+  - 2.4.1
+env:
+  - RAILS_ENV=test
+cache:
+  bundler: true
+services:
+  - postgresql
+addons:
+  apt:
+    packages:
+    - xvfb
+    - postgresql-9.6
+    - postgresql-contrib-9.6
+  postgresql: "9.6"
+before_script:
+  - sudo service postgresql start
+  - cp config/database.yml.travis config/database.yml
+  - psql -c 'create database travis_ci_test;' -U postgres
+  # Install dependencies.
+  - sudo apt-get update
+  - sudo apt-get install -y openjdk-8-jre-headless xvfb libxi6 libgconf-2-4
+
+  # Install Chrome.
+  - wget -N https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -P ~/
+  - sudo dpkg -i --force-depends ~/google-chrome-stable_current_amd64.deb
+  - sudo apt-get -f install -y
+  - sudo dpkg -i --force-depends ~/google-chrome-stable_current_amd64.deb
+
+  # Install ChromeDriver.
+  - wget -N http://chromedriver.storage.googleapis.com/2.27/chromedriver_linux64.zip -P ~/
+  - unzip ~/chromedriver_linux64.zip -d ~/
+  - rm ~/chromedriver_linux64.zip
+  - sudo mv -f ~/chromedriver /usr/local/share/
+  - sudo chmod +x /usr/local/share/chromedriver
+  - sudo ln -s /usr/local/share/chromedriver /usr/local/bin/chromedriver
+
+  # Install Selenium.
+  - wget -N http://selenium-release.storage.googleapis.com/3.0/selenium-server-standalone-3.0.1.jar -P ~/
+  - sudo mv -f ~/selenium-server-standalone-3.0.1.jar /usr/local/share/
+  - sudo chmod +x /usr/local/share/selenium-server-standalone-3.0.1.jar
+  - sudo ln -s /usr/local/share/selenium-server-standalone-3.0.1.jar /usr/local/bin/selenium-server-standalone-3.0.1.jar
+
+  - # start your web application and listen on `localhost`
+  - chrome --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+script:
+  - cp .env.example .env
+  - bundle exec rails db:create
+  - bundle exec rails db:structure:load
+  - bundle exec rails db:seed
+  - rake ci
+  - CODECLIMATE_REPO_TOKEN=0a7c2a9ff76974762fbe7151d5e4aa171f8a86682672d0b32d52ae46b10b3f44 bundle exec codeclimate-test-reporter
```

Comment:
Okay, give me please your codeclimate token for markdown images too
https://docs.codeclimate.com/v1.0/docs/finding-your-test-coverage-token